src/js/helpers/cleanData.js

```diff
@@ -20,8 +20,13 @@ export const cleanContacts = contacts => {
     // Yes, it's `contact.contact` because of the way the API returns data
     let cleaned = Object.assign({}, contact.contact);
 
-    if (cleaned.phone) {
-      cleaned.phone = JSON.parse(cleaned.phone);
+    // quick fix re: phone numbers are current inconsiently entered on the backend
+    try {
+      if (cleaned.phone) {
```

Comment:
maybe it makes sense to put this line outside the try block. I don't remember how our contact component handles null/empty for phone vs how it'll handle json with `default: null` or `default: ''`

Reply:
tweaked in: https://github.com/cityofaustin/janis/pull/408/commits/dc59c4a37a7c5b4d9dee3acf9945cc097dfe0046